tests/test_constraints.py

```diff
@@ -3,38 +3,38 @@
 from cpmpy import boolvar, intvar, Model, cpm_array
 from cpmpy.expressions.core import Comparison, Operator
 from cpmpy.expressions.globalconstraints import *
-from cpmpy.solvers import CPM_gurobi, CPM_ortools, CPM_minizinc
+from cpmpy.solvers import CPM_gurobi, CPM_ortools, CPM_minizinc, CPM_pysat
```

Comment:
I'm not sure this file/changes is intended in this pull request?
should be separate I think

Reply:
I used this test set to test the new XOR constraint in all solvers, so that's why it has changed for Pysat as well...